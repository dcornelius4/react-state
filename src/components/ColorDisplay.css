.ColorDisplay {
  width: 4em;
  height: 4em;
}