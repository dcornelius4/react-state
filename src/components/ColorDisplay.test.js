import React from 'react';
import { shallow } from 'enzyme';
import ColorDisplay from './ColorDisplay';

describe('Color Display Component', () => {
  it('renders the display', () => {
    const wrapper = shallow(<ColorDisplay />);
    expect(wrapper).toMatchSnapshot();
  });
});
