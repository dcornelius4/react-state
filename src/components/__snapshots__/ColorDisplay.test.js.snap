// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Color Display Component renders the display 1`] = `
<div
  className="ColorDisplay"
  style={
    Object {
      "backgroundColor": undefined,
    }
  }
/>
`;
